src/integrations/Sendinblue/browser.js

```diff
@@ -0,0 +1,145 @@
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.name = NAME;
+    this.clientKey = config.clientKey;
+    this.contactAttributeMapping = config.contactAttributeMapping;
+    this.sendTraitsInTrack = config.sendTraitsInTrack;
+  }
+
+  loadScript() {
+    const { clientKey } = this;
+    (function () {
+      window.sib = {
+        equeue: [],
+        client_key: clientKey,
+      };
+      window.sendinblue = {};
+      for (
+        var j = ["track", "identify", "trackLink", "page"], i = 0;
+        i < j.length;
+        i++
+      ) {
+        (function (k) {
+          window.sendinblue[k] = function () {
+            var arg = Array.prototype.slice.call(arguments);
+            (
+              window.sib[k] ||
+              function () {
+                var t = {};
+                t[k] = arg;
+                window.sib.equeue.push(t);
+              }
+            )(arg[0], arg[1], arg[2], arg[3]);
+          };
+        })(j[i]);
+      }
+      var n = document.createElement("script"),
+        i = document.getElementsByTagName("script")[0];
+      (n.type = "text/javascript"),
+        (n.id = "sendinblue-js"),
+        (n.async = !0),
+        (n.src = "https://sibautomation.com/sa.js?key=" + clientKey),
+        n.setAttribute("data-loader", LOAD_ORIGIN),
+        i.parentNode.insertBefore(n, i);
+    })();
+  }
+
+  init() {
+    logger.debug("===In init Sendinblue===");
+    this.loadScript();
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded Sendinblue===");
+    return !!window.sendinblue;
+  }
+
+  isReady() {
+    logger.debug("===In isReady Sendinblue===");
+    return !!window.sendinblue;
+  }
+
+  identify(rudderElement) {
+    logger.debug("===In Sendinblue identify===");
+    const { message } = rudderElement;
+    const { email, phone } = getDefinedTraits(message);
+
+    if (!email || !validateEmail(email)) {
+      logger.error("[Sendinblue]:: provided email is invalid");
+      return;
+    }
+
+    if (phone && !validatePhoneWithCountryCode(phone)) {
+      logger.error("[Sendinblue]:: provided phone number is invalid");
+      return;
+    }
+
+    const userTraits = prepareUserTraits(message, this.contactAttributeMapping);
+    window.sendinblue.identify(email, {
+      ...userTraits,
+    });
+  }
+
+  track(rudderElement) {
+    logger.debug("===In Sendinblue track===");
+    const { message } = rudderElement;
+    const { event } = message;
+    if (!event) {
+      logger.error("[Sendinblue]:: event is required for track call");
+      return;
+    }
+
+    if (this.sendTraitsInTrack) {
+      const { email, phone } = getDefinedTraits(message);
+
+      if (email && !validateEmail(email)) {
+        logger.error("[Sendinblue]:: provided email is invalid");
+        return;
+      }
+
+      if (phone && !validatePhoneWithCountryCode(phone)) {
+        logger.error("[Sendinblue]:: provided phone number is invalid");
+        return;
+      }
```

Comment:
This part is repeated with identify section we can move them to a function and use.

Reply:
in Identify call email is also a required field. In track email is not required

Reply:
- Removed the email check from here. Also, removed the email field from traits.Because if we send email in track call then sendinblue will create the new contact with this email.
- For phone number, Validation is required. Because if we send invalid phone number then track call will not show up in sendinblue dashboard.